pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/RealtimeSegmentImpl.java

```diff
@@ -175,6 +184,59 @@ public RealtimeSegmentImpl(Schema schema, int capacity, String tableName, String
     tableAndStreamName = tableName + "-" + streamName;
   }
 
+  private int getColWidth(FieldSpec.DataType dataType) {
```

Comment:
Better to use Byte.size instead of 8. Also, seems these sizes are already available in V1Constants.Numbers.